### PR TITLE
remove assistant maint access

### PR DESCRIPTION
**Pull request by Ghilker**
## About The Pull Request
Title

## Why It's Good For The Game
Most of the station doesn't have and doesn't need this kind of access, opening doors and removing walls is easy and fast. Assistants should not have a glorified "Ahah tiding go BRRR" with an easy maint access. Let's remove it

## Changelog
:cl:
del: remove maint access from assistants
/:cl:

### CODE REVIEW

File: code/modules/jobs/job_types/assistant.dm [hunk 1]
```diff
@@ -19,7 +19,6 @@ Assistant
 /datum/job/assistant/get_access()
 	if(CONFIG_GET(flag/assistants_have_maint_access) || !CONFIG_GET(flag/jobs_have_minimal_access)) //Config has assistant maint access set
```
Comment (Jared-Fogle) [inline code/modules/jobs/job_types/assistant.dm:20]: ^^^^^^

**Comment by Barhandar** [comment]
How about you first, play the game, and second, play the game, and third, if you want to remove assistants remove assistants entirely?

File: code/modules/jobs/job_types/assistant.dm (hunk 1 above)
Comment (Ghilker) [reply code/modules/jobs/job_types/assistant.dm:20]: this way i don't need headmins approval

**Comment by Jared-Fogle** [comment]
This is already a config option--this is a policy discussion that you should speak with the head admins about if you feel passionate about this.

**Pull request closed without merging**